samples/provider-properties/build.gradle.kts

```diff
@@ -29,7 +29,7 @@ open class GreetingPlugin : Plugin<Project> {
 open class GreetingPluginExtension(project: Project) {
 
     private
-    val messageState = project.property<String>()
+    val messageState = project.objects.property<String>()
```

Comment:
:x: Take the opportunity and rename to `messageProperty`?

Reply:
Done